vision_stack/buoy_detect/README.md

```diff
@@ -0,0 +1,24 @@
+BUOY_DETECT
+-----------
+This repository contains the codes that uses ML (Machine Learning) techniques to detect the buoy. This is a combination of 3 parts - 
+<br>
```

Comment:
@pranaypratyush Don't use `<br>` inside markdown. Rather, use two new lines. (You can also add two whitespaces on the previous line)